ntbs-service/Pages/Notifications/OverviewPartials/_ClinicalDetailsOverviewPartial.cshtml

```diff
@@ -21,6 +21,12 @@
             <div class="cell-min-height"> @Model.Notification.NoSampleTakenYesNo </div>
         </nhs-grid-column>
     </nhs-grid-row>
+    <nhs-grid-row classes="notification-overview-details">

```

Comment:
I thought Symptom onset date should only be set if symptomatic was true? That was just how I understood it from sprint planning, might be misremembering